### ISSUE TITLE
Implement getters/setters 

### ISSUE THREAD

**Issue by frapac** [exanauts/ExaPF.jl#59]
To integrate with `ProxAL.jl` (ref https://github.com/exanauts/ProxAL.jl/issues/3), we need to implement proper getters/setters at the API level. The goal is to get/modify the values of the problem inplace, ala JuMP: 
```
value.(model[:Pg]), value.(model[:Qg])
```
where  `model` is a proper ExaPF object. We could do that at different level (level 1: `PowerNetwork`; level 2: `PolarForm`, level3: `AbstractNLPFormulation`). I think we should discuss where it makes more sense to implement the getters/setters. I see two solutions: 
1-  integrate them directly at the 2nd level (`PolarForm`)
2- create a new object `OptimizationModel` that would mimic a `JuMP.Model`. A prototype could be: 
```julia
abstract type AbstractOptimizationModel end
struct OptimizationModel <: AbstractOptimizationModel
    solution::Dict
    evaluator::AbstractNLPEvaluator
end
function getfield(model::OptimizationModel, symbol)
    ...
end
```

A point to discuss is how to remove a line properly in the network.

**Comment by frapac** [comment]
We should also be able to query this values with the API: 
- objective solution 
- dual variables

**Comment by frapac** [comment]
After a discussion with @michel2323 and @AnirudhSubramanyam , we figured out that we should support the modification of active and reactive power loads at each node of a network: 
> ExaPF demand input: Demand is passed through MOI by directly manipulating the optimization problem defined in ProxAL and accessed in ExaPF.

Thus, we should add two setters to modify properly the values of the load inplace. Loads are specified here: 
https://github.com/exanauts/ExaPF.jl/blob/master/src/models/polar/polar.jl#L14L15

I would suggest the following signatures for the getter/setter:
```julia
id_bus # id of the bus 
model # a PolarForm model
pbus = get(model, PS.ActiveLoad(), BusID(id_bus))
set(model, PS.ActiveLoad(), BusID(id_bus), new_pbus)
```

The main difficulty lies in ensuring that `ExaPF` and `ProxAL` use the same indexing for the buses.

**Comment by frapac** [comment]
Solved by #76
